.github/workflows/release.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Bump and Collect Versions
         run: |
           pnpm changeset version
-          echo "BUILD_VERSION=$(sed 's/.*"version": "(.*?)",$/\1/p' packages/fuels/package.json)" >> $GITHUB_ENV
+          echo "BUILD_VERSION=$(sed -nE 's/^\s*"version": "(.*?)",$/\1/p' packages/fuels/package.json)" >> $GITHUB_ENV
```

Comment:
Using a similar approach for all regexes may improve clarity and readability.

Unless I'm missing something else.

```console
$ sed 's/.*"version": "\(.*\)".*/\1/;t;d' package.json
0.22.2
$ sed 's/.*"forcVersion": "\(.*\)".*/\1/;t;d' forc.json
0.30.0
$ sed 's/.*fuel-core:v\(.*\).*/\1/;t;d' Dockerfile
0.14.0
```